### PR TITLE
Update dependencies by replacing vierbergenlars/php-semver by composer/semver

### PR DESCRIPTION
**Pull request by RobinDev**

### CODE REVIEW

**Comment by ToshY** [comment]
@xtrime-ru Could you take a look at this PR?

**Comment by xtrime-ru** [comment]
Hi! Thanks for PR. I will check it today.

**Comment by RobinDev** [comment]
I also upgrade Puppeteer, the commit are a bit noisy.

File: tests/PuphpeteerTest.php [hunk 1]
```diff
@@ -127,7 +127,7 @@ public function check_all_resources_are_supported(string $name)
                 }
             }
         } else {
-            var_dump($resource->getResourceIdentity());
+            // dump($resource->getResourceIdentity());
```
Comment (xtrime-ru) [inline tests/PuphpeteerTest.php]: Was this change made intentionally?

Comment (RobinDev) [reply tests/PuphpeteerTest.php]: I needed it to debug, i can remove it.

Comment (RobinDev) [reply tests/PuphpeteerTest.php]: (before this bunch of commits, there are no var_dump or dump).

Comment (xtrime-ru) [reply tests/PuphpeteerTest.php]: It would be nice if you clean things like this. And Im ready to merge this.

**Comment by xtrime-ru** [comment]
Thank you very much. 
Now i will change name of package, publish it to packagist and tag version.

**Comment by ToshY** [comment]
@xtrime-ru  I'm eagerly awaiting for it to be on packagist 😄

**Comment by xtrime-ru** [comment]
@ToshY We've added [rialto](https://packagist.org/packages/zoon/rialto) and [puphpeteer](https://packagist.org/packages/zoon/puphpeteer) to packagist. 

Will add to npm later, currently there are github urls in package.json.

Please let me know if you have any issues, because we did not tested new version yet :)

image

**Pull request merged**